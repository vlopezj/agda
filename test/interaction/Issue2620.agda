open import Agda.Builtin.Nat

data B {A : Set} : A → Set where
  b : (a : A) → B a

data C {A : Set} : ∀ {a} → B {A} a → Set where
  c : {a : A} → C (b a)

id : ∀ {b} → C b → B 0
id c = {!!}
