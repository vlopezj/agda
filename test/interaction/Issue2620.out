(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue2620 (Issue2620.agda). " t)
(agda2-info-action "*Type-checking*" " Checking Agda.Builtin.Nat (agda-default-include-path/Agda/Builtin/Nat.agda). " t)
(agda2-info-action "*Type-checking*" " Checking Agda.Builtin.Bool (agda-default-include-path/Agda/Builtin/Bool.agda). " t)
(agda2-info-action "*Type-checking*" " Finished Agda.Builtin.Bool. " t)
(agda2-info-action "*Type-checking*" " Finished Agda.Builtin.Nat. " t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : B 0 _A_18 : Set [ at Issue2620.agda:9,14-15 ] _a_19 : _A_18 [ at Issue2620.agda:9,14-15 ] " nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-status-action "")
(agda2-info-action "*Auto*" "No solution found" nil)
((last . 1) . (agda2-goals-action '(0)))
